### DISCUSSION TITLE
Amazon Fire TV Cube "supported"?

### DISCUSSION THREAD

**Discussion by mrbgfromv** [arendst/Tasmota#10414] [Q&A] [answered]
Hi there,

I can't get the cube to pair with my Tasmota devices, so at the moment I'm unsure if the cube is even "supported" by Tasmota.
In (my) theory the cube should act like an echo dot but the amazon app always shows "no devices found".
If I issue weblog 4 on the console on a device, I get

13:46:00 CFG: in Flash gespeichert am FB, zählen 275, Bytes 4096
13:46:07 WIF: Prüfe Verbindung...
13:46:18 UDP: Packet (119)
13:46:19 UDP: Packet (119)
13:46:20 UDP: Packet (119)
13:46:21 UDP: Packet (119)
13:46:27 WIF: Prüfe Verbindung...
13:46:47 WIF: Prüfe Verbindung...
13:47:07 WIF: Prüfe Verbindung...
13:47:09 UDP: Packet (119)
13:47:10 UDP: Packet (119)
13:47:11 UDP: Packet (119)
13:47:12 UDP: Packet (119)
13:47:27 WIF: Prüfe Verbindung...
13:47:47 WIF: Prüfe Verbindung...
13:48:02 CMD: weblog 2`
`

I'm on 9.2.0 and tried on 8.3.1.6, too. I tried both emulation types.

Before opening an issue I would like to ask for opinions. 
Sadly enough I have no other amazon devices to check if there's a general problem in my home automation system.

Frank

**Comment by s-hadinger** [comment]
I believe it's only Echo devices that can connect to Hue devices (emulated by Tasmota). But I can't cross check.

**Comment by Frogmore42** [comment, marked as answer]
This link says Fire devices might not work with gen 1 Hue Hubs: https://www.reddit.com/r/fireTV/comments/8tw630/fire_tv_cube_and_hue_lights/e1bgudg/

So, the emulation that Tasmota does might be gen 1 (not sure).  I use https://github.com/bwssytems/ha-bridge and it is still working fine, but I have many old Echos still running.  It is not clear if HA-Bridge emulates a gen 1 or gen 2 hub.

**Reply by s-hadinger** [reply]
As soon as you have an echo device, it will be the echo that does the discovery on the LAN. There is no evidence that Fire TV will do the LAN discovery.
